build.gradle

```diff
@@ -38,7 +38,7 @@ subprojects {
 
   apply plugin: 'net.ltgt.errorprone'
   configurations.errorprone {
-    resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.0.11'
+    resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.0.5'
```

Comment:
How do you prefer to handle it wrt this PR ? Rebase the branch on master or merge master into it ?